src/firebase/login.js

```diff
@@ -0,0 +1,57 @@
+import { getAuth, GoogleAuthProvider, onAuthStateChanged, signInWithPopup } from 'firebase/auth';
+import { query, where, getDocs, collection } from 'firebase/firestore';
+import { db, app } from './firebaseConfig';
+
+export var loggedUser = {};
+export var logUserRegData = {};
```

Comment:
I believe this can be shifted to the authContext, and is not needed here anymore.

Reply:
The userInfo (userData and registration data) are getting updated only when signupwithgoogle function is getting called in NavBar.
and authcontext provides that state and modification function. 
I don't think we can move them in authcontext.

Reply:
done. https://github.com/dscnitrourkela/project-udon/pull/11/commits/82f1ce36fecb3e792e89cfbe0fecfa8f6fa46334